src/main/java/seedu/foodrem/logic/commands/itemcommands/EditCommand.java

```diff
@@ -28,25 +27,13 @@
  * Edits the details of an existing item in FoodRem.
  */
 public class EditCommand extends Command {
-
-    public static final String COMMAND_WORD = CommandWord.EDIT_COMMAND.getValue();
-
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Updates the details of the item identified "
-            + "by the index number used in the displayed item list. "
-            + "Existing values will be overwritten by the input values.\n"
-            + "Parameters: INDEX (must be a positive integer) "
-            + "[" + CliSyntax.PREFIX_NAME + "NAME] "
-            + "[" + CliSyntax.PREFIX_ITEM_QUANTITY + "QUANTITY] "
-            + "[" + CliSyntax.PREFIX_ITEM_UNIT + "UNIT] "
-            + "[" + CliSyntax.PREFIX_ITEM_BOUGHT_DATE + "BOUGHT DATE] "
-            + "[" + CliSyntax.PREFIX_ITEM_EXPIRY_DATE + "EXPIRY DATE]...\n"
-            + "Example: " + COMMAND_WORD + " 1 "
-            + CliSyntax.PREFIX_ITEM_QUANTITY + "1000 "
-            + CliSyntax.PREFIX_ITEM_QUANTITY + "grams";
-
-    public static final String MESSAGE_EDIT_ITEM_SUCCESS = "Edited Item: %1$s";
+    // Workaround checkstyle violation
+    // TODO: FIX ME: Make this implementation detail private
     public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
```

Comment:
Why is this public? Is it for testing?

Reply:
On a small note, I'm wondering if there's a better way to keep track of all these static String fields. Reason being, in the future as we add more validation and more meaningful error messages (e.g. Right now we only have an error message for when there's a duplicate item. What if we want to add 3 more error messages?). Just food-for-thought, not sure if we'll ever need to have some Class to handle these messages.

Reply:
Yes, I think this approach is good since it allows us to check and ensure (when we update the test) that the output we get is what we want. Although it may be slightly troublesome to declare another private variable for these messages in the test cases again, ultimately, this will be good in ensuring we get our desired output (and we can use this to further cross-validate and match what we wrote for our command outputs in our UG as well)